ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml

```diff
@@ -4,7 +4,44 @@ postsubmits:
     always_run: true
     branches:
     - ^master$
-    cluster: build02
+    cluster: build04
+    max_concurrency: 1
+    name: branch-ci-openshift-ci-tools-master-build-ci-operator-arm64
+    spec:
+      containers:
+      - args:
+        - --kubeconfig=/etc/build-farm-credentials/kubeconfig
+        - --context=arm01
+        - --namespace=ci
+        - start-build
+        - --follow
+        - --wait
+        - buildconfig/ci-operator-arm64
+        command:
+        - oc
+        image: image-registry.openshift-image-registry.svc:5000/ocp/4.13:cli
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /etc/build-farm-credentials
+          name: build-farm-credentials
+          readOnly: true
+      serviceAccountName: config-updater
+      volumes:
+      - name: build-farm-credentials
+        secret:
+          items:
+          - key: sa.config-updater.arm01.config
+            path: kubeconfig
+          secretName: config-updater
```

Comment:
Are you sure that this is the correct credentials to trigger the build? I am not sure if the `config-updater` has permission on buildConfigs.

Reply:
Yeah it's the account I'm using to do some tests before opening the PRs as mine doesn't have any power there. Give it a shoot eventually, so to have a double check.

Reply:
You need to use the `origin-release-images-builder` service account.

Reply:
I agree, but shouldn't then we
- create `origin-release-images-builder` on `arm01` as well as it exists on `build01` only (check [here](https://github.com/openshift/release/blob/dc969377f33de08e69553cee297c0575b592daa7/clusters/build-clusters/01_cluster/ci/admin_origin-release-images-builder_rbac.yaml))?
- and then force the postsubmit of this PR to always run on `arm01` with the SA set to `origin-release-images-builder` as you're suggesting?